yt_dlp/extractor/firsttv.py

```diff
@@ -16,7 +17,7 @@
 class FirstTVIE(InfoExtractor):
     IE_NAME = '1tv'
     IE_DESC = 'Первый канал'
-    _VALID_URL = r'https?://(?:www\.)?(?:sport)?1tv\.ru/(?:[^/?#]+/)+(?P<id>[^/?#]+)'
+    _VALID_URL = r'https?://(?:www\.)?(?:sport)?1tv\.ru/(?:(?:[^/?#]+/)*(?:(?P<live>live)|(?P<id>[^/?#]+)))'
```

Comment:
Shouldn't this just be a different extractor if we have a different URL structure and none of the code is shared?